notebooks/2023-01-23 Differential abundance testing.Rmd

```diff
@@ -232,9 +274,14 @@ efit_deqms <- suppressWarnings(spectraCounteBayes(efit))
 
 ```
 
+A better relationship, even if it is still weak. Only real difference is higher variance for very low minimum peptides.
+
 ```{r}
 # Diagnostic plots
-VarianceBoxplot(efit_deqms, n = 30, xlab = "Peptides")
+VarianceBoxplot(efit_deqms,xlab = "Peptides")
+VarianceScatterplot(efit_deqms, xlab = "log2(Peptides)", )
+
+log2(18)
```

Comment:
Previously, there was a question why there was a peak in the variance for proteins with 18 peptides. The scatterplot above shows each individual protein, with x-axis on log2 scale. Very few proteins, so difference is likely just chance